ui/components/compose.jsx

```diff
@@ -232,10 +233,18 @@ module.exports = function(core, config, store) {
 		},
 
 		getPlaceholder: function(connection) {
+
 			if (connection === "connecting") {
 				return "Connecting...";
 			} else if (connection === "online") {
-				return "Reply as " + userUtils.getNick(store.get("user")) + ", markdown supported";
+				let roomObj = store.getRoom(),
+					writeLevel = roomObj.guides.authorizer.writeLevel;
+				if(store.isRoomWritable()){
+					return "Reply as " + userUtils.getNick(store.get("user")) + ", markdown supported";
+				} else {
+					return (writeLevel === "follower"? "Follow this room" : "Sign in to scrollback")+ " to send messages";
```

Comment:
Check (user.isGuest() ? "Sign into scrollback" : "Follow this room") instead fo writelevel